docs/documentation/upgrading/topics/changes/changes-26_5_0.adoc

```diff
@@ -36,6 +36,13 @@ the new `UserAuthenticationIdentityProvider` interface (all methods remain the s
 
 Notable changes may include internal behavior changes that prevent common misconfigurations, bugs that are fixed, or changes to simplify running {project_name}.
 
+=== `session_state` and `sid` are no longer UUIDs
+
+In OpenID connect, there are several places where the protocol shares a `session_state` and a `sid`.
+The specifications define it as an opaque string.
+Previous versions of {project_name} used a UUID for it, while the current version now uses a random base64-encoded string.
+The length of the string was reduced from 36 characters to 24 characters, although it might increase in the future if additional randomness might be required.
```

Comment:
```suggestion
The length of the string was reduced from 36 characters to 24 characters, although it might increase in the future if additional randomness is required.
```